src/modules/client/pages/ni/courses/Dashboard.vue

```diff
@@ -165,6 +178,15 @@ export default {
         NotifyNegative('Erreur lors de la récupération des données.');
       }
     },
+    async refreshCompanyLinkRequests () {
+      try {
+        this.linkRequests = await CompanyLinkRequests.list();
+        NotifyPositive('Données mises à jour.');
+      } catch (e) {
```

Comment:
il faut que tu reinitialise `this.linkRequests`

Reply:
done